docs/quickstart/objective-c.md

```diff
@@ -55,7 +55,7 @@ Quickstart. Copy the source code from Github
 [repository](https://github.com/grpc/grpc):
 
 ```sh
-$ git clone https://github.com/grpc/grpc.git
+$ git clone -b {{ site.data.config.grpc_release_branch }} https://github.com/grpc/grpc.git
```

Comment:
Similarly, I suggest adding `--recursive` here and removing the line in the following section that clones the submodules.